projects/aca-content/src/lib/components/search/search-results/search-results.component.scss

```diff
@@ -3,6 +3,10 @@
 .adf-search-results {
   @include flex-row;
 
+  .aca-search-toolbar-spacer {
```

Comment:
Hmm this one seems incorrect. You put .aca-search-toolbar-spacer inside .adf-search-results but if you check html aca-search-toolbar-spacer is outside of adf-search-results

<img width="611" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/d2cf5626-4933-4766-96d4-82e42586744a">


You can use for example: 'aca-search-results' selector which is component's selector instead.

Reply:
Okay, introduced a dedicated scope for the component. Later we can see if we need to move all other styles into the scope too

```scss
aca-search-results {
  .aca-search-toolbar-spacer {
    width: 100%;
  }
}
```